templates/layouts/default.twig

```diff
@@ -1,65 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Not sure about this one, @BackEndTea:

```
$ git grep "layouts/default.twig"
```

yields

```
resources/views/dashboard.twig:1:{% extends "layouts/default.twig" %}
resources/views/error/401.twig:1:{% extends "layouts/default.twig" %}
resources/views/error/403.twig:1:{% extends "layouts/default.twig" %}
resources/views/error/404.twig:1:{% extends "layouts/default.twig" %}
resources/views/error/500.twig:1:{% extends "layouts/default.twig" %}
resources/views/home.twig:1:{% extends "layouts/default.twig" %}
resources/views/ideas.twig:1:{% extends "layouts/default.twig" %}
resources/views/layouts/admin.twig:1:{% extends "layouts/default.twig" %}
resources/views/layouts/reviewer.twig:1:{% extends "layouts/default.twig" %}
resources/views/package.twig:1:{% extends "layouts/default.twig" %}
resources/views/talk/create.twig:1:{% extends "layouts/default.twig" %}
resources/views/talk/edit.twig:1:{% extends "layouts/default.twig" %}
resources/views/talk/view.twig:1:{% extends "layouts/default.twig" %}
resources/views/user/change_password.twig:1:{% extends "layouts/default.twig" %}
resources/views/user/edit.twig:1:{% extends "layouts/default.twig" %}
```

Reply:
Those link towards ```resources/layouts/default.twig``` not ```templates/layouts/default.twig```